.golangci.yml

```diff
@@ -24,6 +24,7 @@ linters:
     # This isn't a widely accepted Go best practice, and would be laborious to
     # apply to existing code.
     - wsl
+    - wsl_v5
```

Comment:
any way to undo all the mess it made?

Reply:
You could probably try to revert most of https://github.com/crossplane/crossplane/pull/6584, but I don't think it's worth it.

Reply:
agreed, i think a lot of folks have already rebased onto all those whitespaces changes, i wouldn't undo them at this point :)